### PR TITLE
Feature translations closes #8

### LINKED ISSUE #8: Add translations
Template is currently hardcoded in English, let's make it possible to support multiple languages.

- [x] Configure everything translatable with `{{ __() }}` function (i.e. login and register screens, etc)
- [x] Create Dutch i18n file
- [x] Update `make:coreui` command to publish translations too

https://laravel.com/docs/5.7/localization

### PR DESCRIPTION
**Pull request by dwaard**

### CODE REVIEW

File: resources/lang/nl/coreui.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    'i_already_have_a_membership' => 'Ik heb al een account',
+    'sign_in'                     => 'Inloggen',
+    'log_out'                     => 'Uitloggen',
+    'toggle_navigation'           => 'Schakel navigatie',
+    'login_message'               => 'Log in om je sessie te starten',
+    'register_message'            => 'Registreer een nieuw account',
+    'password_reset_message'      => 'Wachtwoord herstellen',
+    'reset_password'              => 'Wachtwoord herstellen',
+    'send_password_reset_link'    => 'Stuur email met link voor wachtwoordherstel',
+    'enter_email_message'         => 'Vul uw email addres in',
```
Comment (Lucky-Loek) [inline resources/lang/nl/coreui.php]: Typo: "Vul uw emailadres in"

File: resources/views/auth/register.blade.php [hunk 2]
```diff
@@ -27,51 +27,63 @@
...
+                                <button class="btn btn-primary px-4"
+                                        type="submit">{{ __('coreui::coreui.register') }}</button>
+                            </div>
+                            <div class="col-8 text-right">
+                                <a class="btn btn-link px-0"
+                                   href="{{ route('login') }}">{{ __('coreui::coreui.i_already_have_a_membership') }}</a>
+                            </div>
                         </div>
-                    </div>
+                    </form>
```
Comment (Lucky-Loek) [inline resources/views/auth/register.blade.php:86]: Nice!

File: src/ServiceProvider.php [hunk 3]
```diff
@@ -88,4 +89,17 @@ private function registerViewComposers(Factory $view): void
     {
         $view->composer('coreui::master', CoreUIComposer::class);
     }
+
+    private function loadTranslations()
```
Comment (Lucky-Loek) [inline src/ServiceProvider.php:93]: Nitpicking: it's called `loadTranslations` while it both `loads` and `publishes`. I did the same with `publishConfig()`, but maybe there's a nicer way to fix this. Good for now.

File: resources/lang/nl/coreui.php (hunk 1 above)
Comment (dwaard) [reply resources/lang/nl/coreui.php]: https://onzetaal.nl/taaladvies/website-e-mailadres-web-site-e-mail-adres/

**Pull request merged**